packages/webapp/src/members/components/member-collections.tsx

```diff
@@ -37,33 +37,27 @@ export const MemberCollections = ({ account, templateId }: Props) => {
     }, [account, templateId, tab]);
 
     return (
-        <div className="px-5 py-5 mx-auto flex justify-around">
-            <div className="bg-white rounded-lg p-8 w-full mt-0 md:mt-0 shadow-md">
-                <Button
-                    color="gray"
-                    outline={tab !== "collection"}
-                    disabled={tab === "collection"}
-                    onClick={() => setTab("collection")}
-                >
-                    Collection
-                </Button>
-                <Button
-                    color="gray"
-                    outline={tab !== "collectedBy"}
-                    disabled={tab === "collectedBy"}
-                    onClick={() => setTab("collectedBy")}
-                    className="ml-4"
-                >
-                    Collected By
-                </Button>
-                <hr className="m-2" />
-                {isLoading ? (
-                    "loading..."
-                ) : (
-                    <MembersGrid members={members || []} />
-                )}
-            </div>
-        </div>
+        <Card>
+            <Button
+                color="gray"
+                outline={tab !== "collection"}
+                disabled={tab === "collection"}
+                onClick={() => setTab("collection")}
+            >
+                Collection
+            </Button>
+            <Button
+                color="gray"
+                outline={tab !== "collectedBy"}
+                disabled={tab === "collectedBy"}
+                onClick={() => setTab("collectedBy")}
+                className="ml-4"
+            >
+                Collected By
+            </Button>
```

Comment:
Although I implemented this as buttons, ideally it should be implemented with Tabs. It was just a rush attempt to build these collections cards.

Reply:
Makes sense. I'm in "rush mode" too. Maybe after our first release, we can take a few days to clean up things like this.